docs/rules/index.md

```diff
@@ -5,42 +5,45 @@ layout: doc
 <!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->
```

Comment:
Is there concern re the jekyll support if I've tested it locally and ensured it's working? Pretty sure we can use some CSS hackery to put the icons in, but I felt like this approach would be easier for everyone involved.